docs/_articles/en/user-guide/byotemplate.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Custom Templates
```

Comment:
Do we want to make this title more descriptive. Like: Custom Templates for Custom Code? So folks know why they'd care. Or "Custom Code Templates" ??